### PR TITLE
export _tidb_rowid if exists

### PR DESCRIPTION
**Pull request by holys**

### CODE REVIEW

**Comment by holys** [comment]
reference to: https://github.com/pingcap/tidb/pull/5984

**Comment by holys** [comment]
@GregoryIan  @iamxy  @WangXiangUSTC  PTAL

File: mydumper.c [hunk 1]
```diff
@@ -2759,8 +2761,23 @@ guint64 dump_table_data(MYSQL * conn, FILE *file, char *database, char *table, c
 	GString* statement = g_string_sized_new(statement_size);
 	GString* statement_row = g_string_sized_new(0);
 	
+	query_tidb_rowid = g_strdup_printf("SELECT `_tidb_rowid` FROM `%s`.`%s` LIMIT 0", database, table);
+	no_tidb_rowid = (mysql_query(conn, query_tidb_rowid) || !(result = mysql_use_result(conn)));
+	if (no_tidb_rowid) {
+		if (mysql_errno(conn) == 1054) {
```
Comment (GregoryIan) [inline mydumper.c]: add comment for 1054, `unknown column 'id' in 'field list`

File: mydumper.c [hunk 2]
```diff
@@ -2759,8 +2761,23 @@ guint64 dump_table_data(MYSQL * conn, FILE *file, char *database, char *table, c
 	GString* statement = g_string_sized_new(statement_size);
 	GString* statement_row = g_string_sized_new(0);
 	
+	query_tidb_rowid = g_strdup_printf("SELECT `_tidb_rowid` FROM `%s`.`%s` LIMIT 1", database, table);
```
Comment (WangXiangUSTC) [inline mydumper.c]: how about use a variable save "_tidb_rowid"

**Comment by WangXiangUSTC** [comment]
LGTM

File: mydumper.c (hunk 2 above)
Comment (GregoryIan) [reply mydumper.c]: no need

Comment (holys) [reply mydumper.c]: Should I revert ?

**Comment by holys** [comment]
@GregoryIan  PTAL again

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**